host/3.0/core-tools-publish.yml

```diff
@@ -464,3 +519,69 @@ stages:
         displayName: push python images
         continueOnError: false
         condition: ne(variables['SkipProduction'], 'true')
+
+      - bash: |
+          set -e
+          docker pull $SOURCE_REGISTRY/powershell:$(PrivateVersion)-powershell6-core-tools
+          docker pull $SOURCE_REGISTRY/powershell:$(PrivateVersion)-powershell7-core-tools
+    
+          docker tag $SOURCE_REGISTRY/powershell:$(PrivateVersion)-powershell6-core-tools $TARGET_REGISTRY/powershell:$(TargetVersion)-powershell6-core-tools
+          docker tag $SOURCE_REGISTRY/powershell:$(PrivateVersion)-powershell7-core-tools $TARGET_REGISTRY/powershell:$(TargetVersion)-powershell7-core-tools
+               
+          docker tag $SOURCE_REGISTRY/powershell:$(PrivateVersion)-powershell6-core-tools $TARGET_REGISTRY/powershell:$(MajorVersion)-powershell6-core-tools
+          docker tag $SOURCE_REGISTRY/powershell:$(PrivateVersion)-powershell7-core-tools $TARGET_REGISTRY/powershell:$(MajorVersion)-powershell7-core-tools
+
+        displayName: tag powershell images
+        continueOnError: false
+      - bash: |
+          set -e 
+          PrivateImageId=`docker image inspect $SOURCE_REGISTRY/powershell:$(PrivateVersion)-powershell6-core-tools --format='{{.Id}}'`
+          MajorImageId=`docker image inspect $TARGET_REGISTRY/powershell:$(MajorVersion)-powershell6-core-tools --format='{{.Id}}'`
+          if [[ "$PrivateImageId" != "$MajorImageId" ]]; then
+            echo "unmatch the Target image id and Major image id ${PrivateImageId} and ${MajorImageId}";
+            exit 1;
+          fi
+          PrivateImageId=`docker image inspect $SOURCE_REGISTRY/powershell:$(PrivateVersion)-powershell7-core-tools --format='{{.Id}}'`
+          MajorImageId=`docker image inspect $TARGET_REGISTRY/powershell:$(MajorVersion)-powershell7-core-tools --format='{{.Id}}'`
+          if [[ "$PrivateImageId" != "$MajorImageId" ]]; then
+            echo "unmatch the Target image id and Major image id ${PrivateImageId} and ${MajorImageId}";
+            exit 1;
+          fi
+          ActualVersion=`docker run $TARGET_REGISTRY/powershell:$(MajorVersion)-powershell6-core-tools func --version`
+          if [[ "$ActualVersion" != "$(TargetVersion)" ]]; then
+            echo "unmatch the ActualVersion and TargetVersion ${ActualVersion} and $(TargetVersion)";
+            exit 1;
+          fi
+          docker run $TARGET_REGISTRY/powershell:$(MajorVersion)-powershell6-core-tools az --version
+
+          AZ_MODULE_COUNT=`docker run $TARGET_REGISTRY/powershell:$(MajorVersion)-powershell6-core-tools pwsh  -Command "(Get-Module -ListAvailable Az).count"`
+          if [ ! "$AZ_MODULE_COUNT" -gt "0" ]; then
+            echo "can not find Az Module for powershell6-core-tools. The number of the AzModule is $AZ_MODULE_COUNT"
```

Comment:
Cosmetics: "Az module" ("module" is just a common noun here, no need to capitalize it or merge with "Az").

Reply:
Good catch! I'd like to fix it now.

Reply:
Done!